circle.yml

```diff
@@ -8,14 +8,22 @@ machine:
 
```

Comment:
I was creating some artefacts here :https://github.com/apiaryio/console-proxy/pull/7/files#diff-29944324a3cbf9f4bd0162dfe3975d88R3.

I am totally ok with that, if you think it would be beneficial.